graph-subscriptions-rs/src/subscription_tier.rs

```diff
@@ -0,0 +1,29 @@
+use serde::Deserialize;
+use serde_with::{serde_as, DisplayFromStr};
+
+#[derive(Debug, Default)]
+pub struct SubscriptionTiers(Vec<SubscriptionTier>);
+
+#[serde_as]
+#[derive(Clone, Debug, Default, Deserialize)]
+pub struct SubscriptionTier {
+    /// Payment rate from the subcription contract.
+    #[serde_as(as = "DisplayFromStr")]
+    pub payment_rate: u128,
+    /// Maximum query rate allowed, in queries per second.
+    pub query_rate_limit: u32,
+}
+
+impl SubscriptionTiers {
+    pub fn create(mut tiers: Vec<SubscriptionTier>) -> &'static Self {
```

Comment:
A general constructor like this is typically called `new` ([style guide](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html))
```suggestion
    pub fn new(mut tiers: Vec<SubscriptionTier>) -> &'static Self {
```